README.md

```diff
@@ -291,7 +291,7 @@ backend {
 }
 ```
 
-Or, via `-Dbackend.abortJobsOnTerminate=true` command line option.
+Or, via `-Dserver.abortJobsOnTerminate=true` command line option.
```

Comment:
Ironically, I suspect that `run` mode is far more likely to use this than `server`!